src/Promitor.Agents.ResourceDiscovery/Graph/ResourceTypes/TrafficManagerProfileDiscoveryQuery.cs

```diff
@@ -0,0 +1,21 @@
+using GuardNet;
+using Newtonsoft.Json.Linq;
+using Promitor.Core.Contracts;
+using Promitor.Core.Contracts.ResourceTypes;
+
+namespace Promitor.Agents.ResourceDiscovery.Graph.ResourceTypes
+{
+    public class TrafficManagerProfileDiscoveryQuery : ResourceDiscoveryQuery
```

Comment:
While this might be the metric namespace; I'd suggest to rename everything to `TrafficManager` (in all clases/enum).
```suggestion
    public class TrafficManagerDiscoveryQuery : ResourceDiscoveryQuery
```

That might be more tangible for end-users.